internal/services/registry/storage.go

```diff
@@ -64,28 +70,28 @@ func (p *proc) Handle(ctx context.Context, data *ceData) error {
 	}
 
 	if mtr.Status != models.MetaTransactionRequestStatusConfirmed {
-		return nil
+		return tx.Commit()
 	}
 
-	vehicleMintedEvent := p.ABI.Events["VehicleNodeMinted"]
+	vehicleNodeMintedWithDeviceDefinition := p.ABI.Events["VehicleNodeMintedWithDeviceDefinition"]
 	syntheticDeviceMintedEvent := p.ABI.Events["SyntheticDeviceNodeMinted"]
 
 	switch {
 	case mtr.R.MintRequestUserDevice != nil:
 		for _, logs := range data.Transaction.Logs {
-			if logs.Topics[0] == vehicleMintedEvent.ID {
-				var event contracts.RegistryVehicleNodeMinted
-				err := p.parseLog(&event, vehicleMintedEvent, logs)
+			if logs.Topics[0] == vehicleNodeMintedWithDeviceDefinition.ID {
```

Comment:
Could we handle both `VehicleNodeMinted` and `VehicleNodeMintedWithDeviceDefinition`? I feel like that might make our upgrade path smoother.